### DISCUSSION TITLE
Unicode curly quotes in LaTeX output

### DISCUSSION THREAD

**Discussion by bpj** [jgm/pandoc#10119] [Q&A] [answered]
Please remind me as I never seem able to remember this: is there a way to get Unicode curly quotes in LaTeX output when there are straight quotes in the markdown input without a filter?

**Comment by alerque** [comment, marked as answer]
Disable the (default enabled) *smart* extension on the latex output format:

```console
$ pandoc -t latex-smart <<< "foo's bar"
foo’s bar
```
